app/services/internal/remove_stale_redis_keys_service.rb

```diff
@@ -0,0 +1,10 @@
+class Internal::RemoveStaleRedisKeysService
+  pattr_initialize [:account_id!]
+
+  def perform(account_id)
```

Comment:
The parameter is not required here because the object is initialized with the account_id. That’s why the error “wrong number of arguments (given 0, expected 1)” is thrown.

Also, I’d like to know how the PRESENCE_DURATION and presence_key are initialized in this class. It doesn't seem to inherit or include other methods or classes. AFAIK, the PRESENCE_DURATION is not a global variable. Did we miss some files in this PR?

Reply:
@pranavrajs Thank you for catching this. Fixing this in https://github.com/chatwoot/chatwoot/pull/10562